canonical/vanilla-framework
# Rename list dots to bullets

## Done

Search and replace every instance of dots with bullets

Fixes [list issues/bugs if needed]

## QA

- Open [demo](insert-demo-url)
- make sure there is no more mention of `is-dotted`
- tests pass


### Check if PR is ready for release

If this PR contains Vanilla SCSS code changes, it should contain the following changes to make sure it's ready for the release:

  - `Feature 🎁`, `Breaking Change 💣`, `Bug 🐛`, `Documentation 📝`, `Maintenance 🔨`.
  - if CSS class names are not changed it can be bugfix relesase (x.x.**X**)
  - if CSS class names are changed/added/removed it should be minor version (x.**X**.0)
  - see the [wiki for more details](https://github.com/canonical/vanilla-framework/wiki/Release-process#pre-release-tasks)
